modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -109,6 +110,19 @@ public void transportClientClusterHealth() {
         }
     }
 
+    @Test
+    public void waitingForShards() throws IOException{
+        try (ElasticsearchContainer container = new ElasticsearchContainer(new ImageFromDockerfile()
+            .withFileFromClasspath("data.zip", "elasticsearch-preload/data.zip")
```

Comment:
This is not a common practice. Test should generate its own data IMO - if test data needs fixing , you can fixing it and see your change in same repo. If in your test case you need loads of data to make shard init time high, then I would make wait time configurable and test ES init fail on timeout of 3sec.

Reply:
If I just write data using curl, as an example, I am not testing the waiting for shards, because a clean container does not have any shards, is empty. You need to start a container that already has data loaded.